Resources/views/Sitemap/index.html.twig

```diff
@@ -1,7 +1,7 @@
 <ul class="cmf-sitemap">
     {% for url in urls %}
         <li>
-            <a href="{{ url.location }}" title="{{ url.label }}">{{ url.label }}</a>
+            {% if url.depth is not null %} <span class="indent-{{ url.depth }}"></span>{% endif %}<a href="{{ url.location }}" title="{{ url.label }}">{{ url.label }}</a>
```

Comment:
The experts said: 

To semanticaly put it in the right HTML structure we should create one `<ul></ul>` for each block of children with the same parent (and the same depth). The easy way is to put the `indent-X` classes on the `<li>`element.